packages/firefox/firefox-bin.pacscript

```diff
@@ -0,0 +1,30 @@
+name="firefox-bin"
+version="92.0.1"
+maintainer="aranym <aranym1@pm.me>"
+url="https://download-installer.cdn.mozilla.net/pub/firefox/releases/${version}/linux-x86_64/en-US/firefox-${version}.tar.bz2"
+depends="lsb-release libatk1.0-0 libc6 libcairo-gobject2 libcairo2 libdbus-1-3 libdbus-glib-1-2 libfontconfig1 libfreetype6 libgcc-s1 libgdk-pixbuf-2.0-0 libglib2.0-0 libgtk-3-0 libharfbuzz0b libpango-1.0-0 libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb-shm0 libxcb1 libxcomposite1 libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrender1"
+optdepends=("networkmanager: Location detection via available WiFi networks"
+            "libnotify: Notification integration"
+            "pulseaudio: Audio support"
+            "speech-dispatcher: Text-to-Speech"
+            "hunspell-en_US: Spell checking, American English"
+            "xdg-desktop-portal: Screensharing with Wayland")
```

Comment:
I just copied these dependencies from the PKGBUILD. They are under a different name in debian bases OSes. You'll have to find the correct counterparts and changee the optdeps.

Reply:
I'll make a suggestion with the correct dependency names. Wait a sec.

Reply:
Resolved via https://github.com/pacstall/pacstall-programs/pull/348/commits/f06d72d0c3e91f8303e1a2b8c5f324006b4f6565